coralproject/talk
# fix typo in german localization

## What does this PR do?
This PR fixes a type in the .ftl file for DE which caused the organization name to not render correctly for users who had been suspended.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As an admin, configure your tenant to use Deutsch
2. Pick a commenter, and suspend them
3. Log into a stream as that commenter. 
4. Observe the the suspension message contains the name of your organization, rather than `{orginisation}`

## Where any tests migrated to React Testing Library?
No.

## How do we deploy this PR?
No special considerations needed.
